js/app.js

```diff
@@ -1,11 +1,11 @@
 //This is where application config goes
-angular.module('imdbApp', ['ngRoute']);
+angular.module('imdbApp', ['ngRoute', 'imdbServices', 'imdbControllers']);
```

Comment:
excellent job using the two modules for controllers and services david! 

In the future, when you have a bigger application to work with, you could separate modules based on features. A feature module could have its own controllers, its services, etc.